src/modules/partition/core/PartitionCoreModule.cpp

```diff
@@ -686,12 +683,13 @@ PartitionCoreModule::scanForLVMPVs()
         }
     }
 
-    // Update LVM::pvList
-    LvmDevice::scanSystemLVM( physicalDevices );
-
 #ifdef WITH_KPMCOREGT33
```

Comment:
Can you fix this ifdef so that it would work properly with kpmcore 3.3.1 (i.e. old API for kpmcore 3.3.1 but new for kpmcore 4)